### ISSUE TITLE
Calendar is missing focus trap for Screen Reader users (<p-datepicker>)

### ISSUE THREAD

**Issue by geromegrignon** [openng-org/optimus-ui#415]
Focus should remain trapped inside the calendar until a date is selected.
we cannot able to use p-datepicker without using mouse.

Example:
https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/

> Migrated from `primefaces/primeng#18498` — originally by `@KrishnanAmbuli5501` on 2025-06-18

**Comment by geromegrignon** [comment]
*[Originally posted by `@github-actions[bot]` on 2025-12-16T00:21:39.000Z]*

This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!
